x-pack/plugins/serverless_search/public/layout/nav.tsx

```diff
@@ -37,79 +38,61 @@ const navigationTree: NavigationTreeDefinition = {
-                  pathNameSerialized.startsWith(prepend('/app/maps'))
-                );
-              },
-            },
-            {
-              link: 'management:triggersActions',
-              title: i18n.translate('xpack.serverlessSearch.nav.alerts', {
-                defaultMessage: 'Alerts',
-              }),
-            },
-          ],
+          getIsActive: ({ pathNameSerialized, prepend }) => {
+            return (
+              pathNameSerialized.startsWith(prepend('/app/visualize')) ||
+              pathNameSerialized.startsWith(prepend('/app/lens')) ||
+              pathNameSerialized.startsWith(prepend('/app/maps'))
+            );
+          },
         },
         {
-          id: 'content',
-          title: i18n.translate('xpack.serverlessSearch.nav.content', {
-            defaultMessage: 'Content',
+          link: 'management:triggersActions',
+          title: i18n.translate('xpack.serverlessSearch.nav.alerts', {
+            defaultMessage: 'Alerts',
           }),
-          children: [
-            {
-              title: i18n.translate('xpack.serverlessSearch.nav.content.indices', {
-                defaultMessage: 'Index Management',
-              }),
-              link: 'management:index_management',
-              breadcrumbStatus:
-                'hidden' /* management sub-pages set their breadcrumbs themselves */,
-            },
-            {
-              title: i18n.translate('xpack.serverlessSearch.nav.content.pipelines', {
-                defaultMessage: 'Pipelines',
-              }),
-              link: 'management:ingest_pipelines',
-              breadcrumbStatus:
-                'hidden' /* management sub-pages set their breadcrumbs themselves */,
-            },
-          ],
         },
         {
-          id: 'security',
-          title: i18n.translate('xpack.serverlessSearch.nav.security', {
-            defaultMessage: 'Security',
+          title: i18n.translate('xpack.serverlessSearch.nav.content.indices', {
+            defaultMessage: 'Index Management',
           }),
-          children: [
-            {
-              link: 'management:api_keys',
-              breadcrumbStatus:
-                'hidden' /* management sub-pages set their breadcrumbs themselves */,
-            },
-          ],
+          link: 'management:index_management',
+          breadcrumbStatus: 'hidden' /* management sub-pages set their breadcrumbs themselves */,
+          spaceBefore: 'm',
+        },
+        {
+          title: i18n.translate('xpack.serverlessSearch.nav.content.pipelines', {
+            defaultMessage: 'Pipelines',
+          }),
+          link: 'management:ingest_pipelines',
+          breadcrumbStatus: 'hidden' /* management sub-pages set their breadcrumbs themselves */,
+        },
+        {
+          title: CONNECTORS_LABEL,
+          link: 'serverlessConnectors',
+          breadcrumbStatus: 'hidden' /* management sub-pages set their breadcrumbs themselves */,
```

Comment:
should we hide breadcrumbs for connectors?

Reply:
Whoops no, copy-paste error